lib/justice_gov_sk/decrees.rb

```diff
@@ -0,0 +1,26 @@
+module JusticeGovSk
+  module Decrees
+    extend ActiveSupport::Autoload
+    extend JusticeGovSk::Infrastructure
+
+    autoload :ListCrawler
+    autoload :ResourceCrawler
+    autoload :ResourceParser
```

Comment:
neviem ci je autoload vhodny na tomto mieste, pride mi to roztrusene, nieco je navyse duplicitne v `lib/justice_gov_sk.rb`

navrhujem aby sme dali vsetko do `lib/justice_gov_sk.rb` a podla potreby nastavit eager loading